include/arch/arc/v2/linker.ld

```diff
@@ -134,41 +154,64 @@ SECTIONS {
 		 */
 		. = ALIGN(4);
 		__bss_start = .;
-		*(.bss)
-		*(".bss.*")
-		COMMON_SYMBOLS
+
+#ifndef CONFIG_APPLICATION_MEMORY
+		_image_ram_start = .;
+#endif
+		__kernel_ram_start = .;
+		KERNEL_INPUT_SECTION(.bss)
+		KERNEL_INPUT_SECTION(".bss.*")
+		KERNEL_INPUT_SECTION(COMMON)
+		*(".kernel_bss.*")
+
 		/*
 		 * BSP clears this memory in words only and doesn't clear any
 		 * potential left over bytes.
 		 */
 		__bss_end = ALIGN(4);
 	} GROUP_DATA_LINK_IN(RAMABLE_REGION, RAMABLE_REGION)
 
-	SECTION_PROLOGUE(_NOINIT_SECTION_NAME,(NOLOAD),) {
+SECTION_PROLOGUE(_NOINIT_SECTION_NAME,(NOLOAD),) {
```

Comment:
don't think you want to get rid of the leading tab here

Reply:
typo, i will fix it.